rosidl_generator_cpp/resource/srv.hpp.template

```diff
@@ -12,6 +12,7 @@
 #ifndef __@(spec.pkg_name)__srv__@(get_header_filename_from_msg_name(spec.srv_name))__hpp__
 #define __@(spec.pkg_name)__srv__@(get_header_filename_from_msg_name(spec.srv_name))__hpp__
 
+#include "@(spec.pkg_name)/srv/@(get_header_filename_from_msg_name(spec.srv_name))__traits.hpp"
```

Comment:
The `has_bounded_size` trait for services is just a convenience for checking that both the request and response messages for the service are bounded.